File: src/main/java/constants/header/HeaderDropDown.java
package constants.header;

public enum HeaderDropDown {
    LANGUAGE("Language"),
    CURRENCY("Currency"),
    AGENTS("Agents"),
    CUSTOMER("Customer");

    private final String name;

    HeaderDropDown(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }
}

File: ts-playwright-test-automation-framework/src/main/code/constants/header/HeaderDropDown.ts
export class HeaderDropDown {
    static readonly LANGUAGE = new HeaderDropDown("Language");
    static readonly CURRENCY = new HeaderDropDown("Currency");
    static readonly AGENTS = new HeaderDropDown("Agents");
    static readonly CUSTOMER = new HeaderDropDown("Customer");

    private constructor(
        private readonly name: string
    ) { }

    getName(): String {
        return this.name;
    }
}